services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/converter/IofSwitchConverter.java

```diff
@@ -15,18 +32,19 @@
 /**
  * Converter of floodlight switch representation {@link net.floodlightcontroller.core.IOFSwitch}.
  */
-public final class IOFSwitchConverter {
+public final class IofSwitchConverter {
 
     /**
      * Transforms {@link IOFSwitch} to object that is used throughout kilda in all components.
+     *
      * @param sw switch data.
      * @param eventType switch state.
      * @return converted switch.
      */
     public static SwitchInfoData buildSwitchInfoData(IOFSwitch sw, SwitchState eventType) {
-        String switchId = sw.getId().toString();
+        SwitchId switchId = new SwitchId(sw.getId().toString());
```

Comment:
`DatapathId` (you have it from `sw.getId()`), have method `.getLong()` - add constructor into `SwitchId` that accept Long value and it will eliminate such Strings conversations, plus it will simplify SwtichId definitions in test code... You can write `new SwitchId(1L)`, instead of `new SwitchId("00:00:00:00:00:00:00:01")`.

Reply:
This has been fixed.